app/scripts/services/rowMethods.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+angular.module('openshiftConsole')
+  .factory('rowMethods', function() {
```

Comment:
Maybe just expand/collapse? We have some services already saving state in local/session storage like AlertsMessageService